<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>499</width>
                <height>378</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Ingreso de Administrador</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>170</x>
                        <y>0</y>
                        <width>131</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>MENÚ DE CONEXIÓN</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>50</y>
                        <width>251</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>BIENVENIDO A TU SISTEMA GESTOR DE BD</string>
                </property>
            </widget>
            <widget class="QLabel" name="USUARIO">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>90</y>
                        <width>55</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>USUARIO</string>
                </property>
            </widget>
            <widget class="QLabel" name="CONTRASENA">
                <property name="geometry">
                    <rect>
                        <x>70</x>
                        <y>150</y>
                        <width>81</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>CONTRASEÑA</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn_conectar">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>280</y>
                        <width>93</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>CONECTAR</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="le_User">
                <property name="geometry">
                    <rect>
                        <x>50</x>
                        <y>120</y>
                        <width>113</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="le_Contra">
                <property name="geometry">
                    <rect>
                        <x>50</x>
                        <y>180</y>
                        <width>113</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="CONTRASENA_2">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>100</y>
                        <width>31</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>DNS</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="le_Dns">
                <property name="geometry">
                    <rect>
                        <x>300</x>
                        <y>120</y>
                        <width>113</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="CONTRASENA_3">
                <property name="geometry">
                    <rect>
                        <x>340</x>
                        <y>150</y>
                        <width>31</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>HOST</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="le_Host">
                <property name="geometry">
                    <rect>
                        <x>300</x>
                        <y>180</y>
                        <width>113</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="le_Port">
                <property name="geometry">
                    <rect>
                        <x>170</x>
                        <y>240</y>
                        <width>113</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="CONTRASENA_4">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>220</y>
                        <width>41</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Puerto</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>499</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
